src/components/footer/footer.js

```diff
@@ -1,60 +1,91 @@
-import React, {useState} from "react";
+import React, { useState } from "react";
```

Comment:
This entire file, and all others that were edited, need to be reformatted to use tabs set to an indent of 4 instead of spaces. (This was stated as part of our style guide in the very long pinned slack post in dev. Understandable if you missed it at the end)

Reply:
For some reason I cannot figure out how to get the code to refactor on this component. I have changed the tab indent to 4, but it is still double the width. I have added all other changes and am about to push them